gradle/scripts/dependencyDefinitions.gradle

```diff
@@ -162,7 +158,16 @@ ext.externalDependency = [
     "googleOauthClient": "com.google.oauth-client:google-oauth-client:" + googleVersion,
     "googleApiClient": "com.google.api-client:google-api-client:" + googleVersion,
     "opencsv": "com.opencsv:opencsv:3.8",
-    "hadoopAdl" : "org.apache.hadoop:hadoop-azure-datalake:3.0.0-alpha2"
+    "hadoopAdl" : "org.apache.hadoop:hadoop-azure-datalake:3.0.0-alpha2",
+    "slf4j": [
+        "org.slf4j:slf4j-api:" + slf4jVersion,
+        "org.slf4j:slf4j-log4j12:" + slf4jVersion,
+        "org.slf4j:jcl-over-slf4j:" + slf4jVersion
```

Comment:
If you don't have `jcl-over-slf4j` then the jcl logs won't go through the slf4j subsystem and be logged by log4j.